platform-sdk/docs/proposals/README.md

```diff
@@ -0,0 +1,145 @@
+4. Changes to interactions between major systems or teams (platform, services, mirror node, devops, etc)
+5. Module Structure & Repos
+
+---
+
+## Design Proposal Process Flow
+
+![](designProposalFlow.drawio.svg)
+
+## Creating A Proposal
+
+All proposals have an owner who is responsible for taking it through the process to completion. Proposals must be
+presented in the form of a PR containing the proposal in a subdirectory of `platform-sdk/docs/proposals/`. Proposals may
+contain diagrams and supporting materials. Diagrams must be created with DrawIO and exported in SVG format. All content
+must be editable (i.e. no images).
+
+If any APIs were developed in code as part of the design process, it may be included in the proposal directly or in the
+subdirectory of the proposal, but not merged into the source code of the platform.
+
+While in the draft phase the proposal may be updated and modified as needed. It is best practice to solicit review from
+stakeholders to surface any potential change requests while the proposal is in the draft phase. All grammar and spelling
+errors should be caught during this phase.
+
+### Creating a Draft Proposal
+
+1. Create a new branch for the proposal.
+2. Create a subdirectory of `platform-sdk/docs/proposals/` with the name of the proposal.
+3. Copy the template `template.md` into the subdirectory and give it an appropriate name.
+4. Fill out the template with the details of the proposal, removing any irrelevant sections or adding new sections as
+   needed. Diagrams and other files may be included in the proposal directory and linked from the proposal.
+5. Create a draft PR with the proposal documents.
+6. Put the draft PR in the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board in
+   the `Draft` status.
+7. Solicit feedback from stakeholders and other design collaborators.
+8. Iterate on the proposal until it is ready for voting. All open comments should be resolved while in the draft stage.
+
+### Submitting a Proposal for Voting
+
+When the proposal is ready for voting, take the following steps:
+
+1. Ensure all comments are resolved.
+2. Mark the PR as `Ready for Review`
+3. Update the status of the PR in
+   the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board to `Voting`
+
+Requests for immaterial changes can be addressed by pushing additional commits to the existing proposal
+PR while in `Voting`. Examples of immaterial changes are spelling or grammar errors, wording changes that do not modify
+meaning or intent, and formatting changes. Examples of material changes include changes to behaviors or APIs, addition
+or removal of diagrams, etc. If any material changes are needed, the PR must be closed and moved to the `Superseded`
+status. A new proposal PR is prepared and voting is restarted. The `Superseded` PR must reference the new PR to
+create an auditable history.
```

Comment:
-1

This defeats the entire purpose of using a PR to do proposals. It creates a ton of extra paperwork, and clears away the comment history. If we want to do proposals like this, it will need to be by decree of management. I don't think I can be convinced vote for this pattern.

Reply:
I don't agree that it defeats the entire purpose of using a PR for proposals. Using a PR still has benefits:

1. Reviewers can comment on the exact line where you have a question or comment
2. Proposal owners do not have to do double the work to convert a proposal from ticket format to markdown format when the feature is delivered
3. It leverages the built-in feature of "Approve" indicating a +1 vote and a "Changes Request" indicating a -1 vote.

I think it's completely appropriate and good to require a new PR for material changes. As @edward-swirldslabs pointed out in another comment that I can no longer find, the burden of opening a new PR is low enough to not slow down the process too much, but is enough burden to incentivize proposal owners to ensure they have a proposal that has a good chance of being accepted before everyone spends their time formally reviewing it.

Reply:
> the burden of opening a new PR is low enough to not slow down the process too much, but is enough burden to incentivize proposal owners to ensure they have a proposal that has a good chance of being accepted before everyone spends their time formally reviewing it.

Sounds like we're introducing this 'new PR' burden to disincentivize submitting PRs without sufficient due diligence. I think there's already enough burden in PRs not getting past voting with having to iterate the process over and over. We don't need n additional burden. I vote for sticking to the PR workflow where all changes needed to pass a vote occurs on the same PR for two reasons:

1. it's intuitive, well-understood workflow
2. it preserves entire feedback history 
3. PR process will stay consistent

I don't think it's worth introducing two workflows - one for 'code PRs' and another for 'design PRs'.

All of these said, I am not as strongly against it as @cody-littley is :-) Happy to try the new process to get us started.